src/components/card/card.component.spec.ts

```diff
@@ -1,28 +1,84 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { CardComponent } from './card.component';
+import { MatCardModule } from '@angular/material/card';
+import { TabbedChartsMeta } from '../../datasets/metas/tabbed-charts.meta';
+import { LineChartMeta } from '../../datasets/metas/line-chart.meta';
 
 describe('CardComponent', () => {
-  let fixture: ComponentFixture<CardComponent>;
-  let component: CardComponent;
+  let tabbedFixture: ComponentFixture<CardComponent<TabbedChartsMeta>>;
+  let tabbedComponent: CardComponent<TabbedChartsMeta>;
+  let lineFixture: ComponentFixture<CardComponent<LineChartMeta>>;
+  let lineComponent: CardComponent<LineChartMeta>;
+
+  const testChartMeta1: LineChartMeta = {
+    type: 'line',
+    title: 'testChart1',
+    query: () => [],
+  };
+  const testChartMeta2: LineChartMeta = {
+    type: 'line',
+    title: 'testChart2',
+    query: () => [],
+  };
+  const testTabbedChartsMeta: TabbedChartsMeta = {
+    type: 'tabbed',
+    title: 'testTabbedCharts',
+    metas: [testChartMeta1, testChartMeta2],
+  };
```

Comment:
How do you think of moving these to `src/utils/mocks.spec.ts`?

Reply:
I think these are small enough that they should stay here. Makes it easier to reference them when reading the tests.

I would only move test data if it is HUGE or if it is shared by more than one test file

Reply:
ok sounds good. I'll move it once I find another place to use it.